src/next-transpile-modules.js

```diff
@@ -312,6 +311,34 @@ const withTmInitializer = (modules = [], options = {}) => {
           }
         }
 
+        // Add support for Global CSS imports in transpiled modules
+        if (nextCssLoaders) {
+          const nextGlobalCssLoader = nextCssLoaders.oneOf.find(
+            (rule) => rule.sideEffects === true && regexEqual(rule.test, /(?<!\.module)\.css$/)
+          );
+
+          if (nextGlobalCssLoader) {
+            nextGlobalCssLoader.issuer = { or: [matcher, nextGlobalCssLoader.issuer] };
+            nextGlobalCssLoader.include = { or: [...modulesPaths, nextGlobalCssLoader.include] };
+          } else if (!options.isServer) {
+            // Note that Next.js ignores global CSS imports on the server
```

Comment:
What does that mean?

Reply:
We don’t need to modify any rules for the server Webpack configuration, since it always ignores global CSS imports (by default).